src/ui/controls/dropdown/style.less

```diff
@@ -47,7 +47,10 @@
         overflow: hidden;
         position: absolute;
         width: 100%;
-
+         max-height: 10rem;                   // added
```

Comment:
CSS properties should be sorted alphabetically, white space indentation should be the same. No unnecessary comments.

Why did you set 10rem here?